src/pipeline-status-slack.py

```diff
@@ -158,6 +158,12 @@ def lambda_handler(event, context):
     timestamp = event["time"].split(".")[0]
     timestamp = datetime.strptime(timestamp[:-1], "%Y-%m-%dT%H:%M:%S")
     execution_input = json.loads(event["detail"]["input"])
+    additional_slack_webhook_urls = execution_input.get(
+        "slack_webhook_urls", []
+    )
+    slack_webhook_urls = list(
+        set([slack_webhook_url] + additional_slack_webhook_urls)
+    )
```

Comment:
This could be simplified to:

```python
slack_webhook_urls = {slack_webhook_url, *additional_slack_webhook_urls}
```

You don't need to convert it back to list, as the interface is the same for your usecase.

Reply:
Cool, wasn't actually aware of the set notation you used there! I'll just stick with the current syntax for now as I think the braces are a bit more ambiguous wrt. to dicts vs sets 😄 

Old habit of converting to list in case we need indexing, but yeah, not really needed for the current use-case here